db/seeds.rb

```diff
@@ -1,6 +1,21 @@
 Prison.transaction do
+  prison_uuid_mapping_path =
+    Rails.root.join('db', 'seeds', 'prison_uuid_mappings.yml')
+
+  prison_uuid_mappings = YAML.load(File.read(prison_uuid_mapping_path)) || {}
+
   Dir[Rails.root.join('db', 'seeds', 'prisons', '*.yml')].each do |path|
     hash = YAML.load(File.read(path))
+    filename = Pathname.new(path).basename.to_s
+
+    next if Prison.find_by(id: prison_uuid_mappings[filename])
+
+    if prison_uuid_mappings[filename].blank?
+      fail Prison::MissingUuidMapping, <<-EOF
```

Comment:
`String`'s `:strip_heredoc` method can be used to remove indentation on heredocs.

```ruby
  fail Prison::MissingUuidMapping, 
    <<-INDENTATION_NAZI.strip_heredoc
      #{filename} is missing a UUID mapping. Rerun `rake maintenance:prison_uuids`,
      commit the result and rerun `rake db:seed`.
    INDENTATION_NAZI
```